sql/core/src/main/scala/org/apache/spark/sql/execution/ui/SQLAppStatusListener.scala

```diff
@@ -344,6 +344,18 @@ class SQLAppStatusListener(
     update(exec)
   }
 
+  private def onAdaptiveAccumUpdates(event: SparkListenerSQLAdaptiveAccumUpdates): Unit = {
+    val SparkListenerSQLAdaptiveAccumUpdates(executionId, accumIdsToMetricType) = event
+
+    val stages = liveExecutions.get(executionId).stages
+    accumIdsToMetricType.map { case (accumulatorId, metricType) =>
```

Comment:
why do we need to loop it? we can just update `liveStageMetric.accumIdsToMetricType` as
```
liveStageMetric.copy(accumIdsToMetricType =
  liveStageMetric.accumIdsToMetricType ++ accumIdsToMetricType)
```